XMLtoMD/GuidelineXmlToMD/Program.cs

```diff
@@ -0,0 +1,179 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text.RegularExpressions;
+using MarkdownOut;
+
+
+namespace GuidelineXmlToMD
+{
+    class Program
+    {
+        static MarkdownOut.MdWriter _MdWriter;
+        static void Main(string[] args)
+        {
+            string xmlFileName = "Guidelines(8th Edition).xml";
+            if (args.Length != 0) {  //check for input fileName
+                if (Regex.Match(args[0], @".*.xml").Success) {
+                    xmlFileName = args[0];
+                }
+            }
+
+            Match repoRefFolder = Regex.Match(AssemblyDirectory, @$".*CodingGuidelines");
+
+            string[] xmlFilePath = { repoRefFolder.Value, "docs", xmlFileName};
+            ICollection<Guideline> guidelines = GuidelineXmlFileReader.ReadExisitingGuidelinesFile(Path.Combine(xmlFilePath));
```

Comment:
It may be cleaner to simply pass in the path to the xml file as a required argument to Main rather than trying to derive it.